src/helpers/event/recurringEventHelpers/generateRecurringEventInstances.ts

```diff
@@ -51,12 +50,10 @@ export const generateRecurringEventInstances = async ({
 }: InterfaceGenerateRecurringInstances): Promise<InterfaceEvent> => {
   const recurringInstances: InterfaceRecurringEvent[] = [];
   recurringInstanceDates.map((date) => {
-    const formattedInstanceDate = format(date, "yyyy-MM-dd");
-
     const createdEventInstance = {
       ...data,
-      startDate: formattedInstanceDate,
-      endDate: formattedInstanceDate,
+      startDate: date,
+      endDate: date,
```

Comment:
Also make sure abut date formatting here

Reply:
same here, rrule returns the dates in proper format, no need to format it again.